src/sentry/api/endpoints/organization_derive_code_mappings.py

```diff
@@ -48,15 +48,15 @@ def get(self, request: Request, organization: Organization) -> Response:
         installation, _ = get_installation(organization)
         if not installation:
             return self.respond(
-                "Could not find this integration installed on your organization",
+                {"text": "Could not find this integration installed on your organization"},
                 status=status.HTTP_404_NOT_FOUND,
             )
 
         trees = installation.get_trees_for_org()
         trees_helper = CodeMappingTreesHelper(trees)
-        frame_filename = FrameFilename(stacktrace_filename)
-        possible_code_mappings: List[Dict[str, str]] = trees_helper.list_file_matches(
-            frame_filename
+        frame_filename = FrameFilename(stacktrace_filename) if stacktrace_filename else None
+        possible_code_mappings: List[Dict[str, str]] = (
+            trees_helper.list_file_matches(frame_filename) if frame_filename else []
         )
```

Comment:
for these ones, i was unsure whether it was better to deal with cases where an argument is `None` but the function expects it not to be in the call like I did here or in the function itself (thus needing to change the typing so that the argument is `Optional`)

Reply:
+1 to your approach - I think it's better to check in the caller since the function expects a not-None argument.